src/prefect/cli/block.py

```diff
@@ -109,3 +112,56 @@ async def register(
         "\n To configure the newly registered blocks, "
         "go to the Blocks page in the Prefect UI.\n"
     )
+
+
+@blocks_app.command()
+async def ls():
+    """
+    View all configured Blocks.
+    """
+    async with get_client() as client:
+        blocks = await client.read_block_documents()
+
+    table = Table(
+        title="Blocks", caption="List Block Types using `prefect block type ls`"
+    )
+    table.add_column("ID", style="cyan", no_wrap=True)
+    table.add_column("Name", style="blue", no_wrap=True)
+    table.add_column("Type", style="blue", no_wrap=True)
+
+    for block in sorted(blocks, key=lambda x: x.block_schema.fields["title"]):
+        table.add_row(
+            str(block.id),
+            str(block.name),
+            block.block_schema.fields["title"],
```

Comment:
This title field doesn't always match the block type name. I think you should be able to do this instead:
```suggestion
            block.block_type.name,
```